framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -2900,18 +2900,26 @@ private boolean checkReceiverOverride() {
         }
 
         private boolean checkParameters() {
-            boolean result = true;
-            // Check parameter values. (TODO: FIXME varargs)
             List<AnnotatedTypeMirror> overriderParams = overrider.getParameterTypes();
             List<AnnotatedTypeMirror> overriddenParams = overridden.getParameterTypes();
 
-            // The functional interface of an unbound member reference has an extra parameter (the receiver).
-            if (methodReference
-                    && ((JCTree.JCMemberReference) overriderTree)
-                            .hasKind(JCTree.JCMemberReference.ReferenceKind.UNBOUND)) {
-                overriddenParams = new ArrayList<>(overriddenParams);
-                overriddenParams.remove(0);
+            // Fix up method refernece parameters.
```

Comment:
Spell-check.
(I can't make changes directly when the pull request is from a branch that exists only on your own fork.)